HNG_model.py

```diff
@@ -66,15 +72,18 @@ def residual(t, solution):
 #    dconc_dt = - m.tanh(30*(conc-1))*nuclii*(drad_dt*2*m.pi*radius**2)/mol_vol/V_elect/co_k  distrute change in mass to electrolyte
     return [dr_dt, dc_dt]
 
-"========================== Run the simulation =========================="
-solution = solve_ivp(residual, [0, time]], sol_vec) #growth senario
+"""========================== Run the simulation =========================="""
+solution = solve_ivp(residual, [0, time], sol_vec) #growth senario
+
+"""============================ Post-processing ============================"""
+
 
-"============================ Post-processing ============================"
 radius = solution.y[0]
 concentrations = solution.y[1]
 print(concentrations)
 t = solution.t
-plt.figure(0)
+
+fig = plt.figure(0)
```

Comment:
Are these lines redundant with the `PdfPages` implementation below?  I'll admit, I'm not totally clear on what the PdfPages is, so maybe there is a reason to having them both...

Reply:
These are redundant, I just wanted it to print the figure in the code, rather than having to open the file every time.  It was just being lazy, I will take those out of the final version.

Reply:
I think you can just add `plt.show()` right before `plt.close()` on line 116 and it will print all the figures.

Reply:
That worked thanks!